projects/ccd-case-ui-toolkit/src/lib/shared/components/palette/linked-cases/read-linked-cases-field.component.ts

```diff
@@ -0,0 +1,89 @@
+import { AfterViewInit, Component, OnInit } from '@angular/core';
+import { ActivatedRoute, Router } from '@angular/router';
+import { AbstractAppConfig } from '../../../../app.config';
+import { CaseField, CaseTab } from '../../../domain';
+import { CommonDataService } from '../../../services/common-data-service/common-data-service';
+import { LinkedCasesService } from './services';
+
+@Component({
+  selector: 'ccd-read-linked-cases-field',
+  templateUrl: './read-linked-cases-field.component.html',
+  styleUrls: ['./read-linked-cases-field.component.scss']
+})
+export class ReadLinkedCasesFieldComponent implements OnInit, AfterViewInit {
+
+  public caseField: CaseField;
+
+  public reasonListLoaded = false;
+  public reload = false
```

Comment:
@johnbenjamin-hmcts - missing semicolon

Reply:
@Daniel-Lam I have updated the code now.